src/iosrv/init.c

```diff
@@ -211,13 +211,13 @@ dss_tgt_nr_get(int ncores, int nr)
 	 * if user requires more then uses the number calculated above
 	 * as creating more threads than #cores may hurt performance.
 	 */
```

Comment:
probably need to change the above comment a little bit. thx

Reply:
agreed, must be fixed. We also need to fix the first line of the commit to match our standard, i.e. DAOS-2799 iosrv: allow more xstreams than the number of cores